selfdrive/test/longitudinal_maneuvers/maneuver.py

```diff
@@ -30,56 +25,20 @@ def evaluate(self):
       distance_lead=self.distance_lead
     )
 
-    logs = defaultdict(list)
-    last_controls_state = None
-    plot = ManeuverPlot(self.title)
-
-    buttons_sorted = sorted(self.cruise_button_presses, key=lambda a: a[1])
-    current_button = 0
+    valid = True
     while plant.current_time() < self.duration:
-      while buttons_sorted and plant.current_time() >= buttons_sorted[0][1]:
-        current_button = buttons_sorted[0][0]
-        buttons_sorted = buttons_sorted[1:]
-        print("current button changed to {0}".format(current_button))
-
-      grade = np.interp(plant.current_time(), self.grade_breakpoints, self.grade_values)
       speed_lead = np.interp(plant.current_time(), self.speed_lead_breakpoints, self.speed_lead_values)
+      log = plant.step(speed_lead)
 
-      # distance, speed, acceleration, distance_lead, brake, gas, steer_torque, fcw, controls_state= plant.step(speed_lead, current_button, grade)
-      log = plant.step(speed_lead, current_button, grade)
-
-      if log['controls_state_msgs']:
-        last_controls_state = log['controls_state_msgs'][-1]
 
       d_rel = log['distance_lead'] - log['distance'] if self.lead_relevancy else 200.
       v_rel = speed_lead - log['speed'] if self.lead_relevancy else 0.
       log['d_rel'] = d_rel
       log['v_rel'] = v_rel
 
-      if last_controls_state:
-        # print(last_controls_state)
-        #develop plots
-        plot.add_data(
-          time=plant.current_time(),
-          gas=log['gas'], brake=log['brake'], steer_torque=log['steer_torque'],
-          distance=log['distance'], speed=log['speed'], acceleration=log['acceleration'],
-          up_accel_cmd=last_controls_state.upAccelCmd, ui_accel_cmd=last_controls_state.uiAccelCmd,
-          uf_accel_cmd=last_controls_state.ufAccelCmd,
-          d_rel=d_rel, v_rel=v_rel, v_lead=speed_lead,
-          v_target_lead=last_controls_state.vTargetLead, pid_speed=last_controls_state.vPid,
-          cruise_speed=last_controls_state.vCruise,
-          a_target=last_controls_state.aTarget,
-          fcw=log['fcw'])
-
-        for k, v in log.items():
-          logs[k].append(v)
-
-    valid = True
-    for check in self.checks:
-      c = check(logs)
-      if not c:
-        print(check.__name__ + " not valid!")
-      valid = valid and c
+    if d_rel < 1.0:
```

Comment:
was this intentional? only check at the end? if so, let's also remove the unused checks in `test_longitudinal.py`

Reply:
No it was not. it should check every iteration

Reply:
if we're removing the fcw check, should we remove the fcw test cases? https://github.com/commaai/openpilot/blob/simplify_longitudinal_tests/selfdrive/test/longitudinal_maneuvers/test_longitudinal.py#L31

Reply:
You can remove them, but put a todo to make new ones. I want to make new fcw and test that accordingly with this test.